src/main/java/com/ibm/nfvodriver/service/LifecycleManagementService.java

```diff
@@ -40,77 +47,60 @@ public LifecycleManagementService(NSLifecycleManagementDriver nsLifecycleManagem
 
     public ExecutionAcceptedResponse executeLifecycle(ExecutionRequest executionRequest) throws MessageConversionException {
         logger.info("Processing execution request");
-
+        String lifecycleName = executionRequest.getLifecycleName();
+        final String requestId = UUID.randomUUID().toString();
+        String requestPayload = null;
+        if(requestTypeMap.containsKey(lifecycleName))
```

Comment:
Put an else condition and throw an exception saying not supported lifecycle name.